php/resources/root/docker-entrypoint.sh

```diff
@@ -4,55 +4,22 @@ set -e
 # exit when docker sends SIGTERM (aka docker stop)
 onTerminate() {
     echo "Got SIGTERM, stopping apache..."
-    kill `cat /run/apache2/apache2.pid`
+    kill `cat /var/run/httpd/httpd.pid`
     exit
 }
 trap 'onTerminate' SIGTERM
 
-#
```

Comment:
This code is still necessary and the other removed code below will still need to be enabled at some point.

If this is causing problems, I'd prefer it move to a separate entrypoint instead of being deleted.

Reply:
These MySQL settings can be set up via command line or via `docker-compose.yml`. Importing the dumps can be used by mounting the dir they are in to `/docker-entrypoint-initdb.d`.

Also, this container should be able to stand on its own, so we should not have a dependency on the MySQL container directly within this container.